packages/components/src/color-palette/index.js

```diff
@@ -135,7 +137,12 @@ export default function ColorPalette( {
 		<VStack spacing={ 3 } className={ className }>
 			{ ! disableCustomColors && (
 				<Dropdown
-					contentClassName="components-color-palette__custom-color-dropdown-content"
+					contentClassName={ classnames(
```

Comment:
Could we just pass down a css class instead of introducing a new API?

Reply:
We would need add a property like customColorContentClassName, to ColorPalette, and ColorGradientPanels etc. A block wanting to have the color picker appear in the correct position would need to explicitly know the class to add.
It seems preferable to have a flag that changes the positioning behavior. The fact we use a class to do that is an implementation detail, e.g: on mobile, the way to do it may be different.